tmail-backend/jmap/extensions/pom.xml

```diff
@@ -19,6 +19,12 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>team-mailboxes</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-backends-es-v7</artifactId>
```

Comment:
Not found of that... However I need that to include QueryBuilder in the contract to test properly update with ES implem. I could not find any better... But I'm opened for ideas

Reply:
we can add only elasticsearch client dependency?
```
        <dependency>
            <groupId>org.elasticsearch.client</groupId>
            <artifactId>elasticsearch-rest-high-level-client</artifactId>
            <version>7.10.2</version>
            <exclusions>
                <exclusion>
                    <groupId>commons-logging</groupId>
                    <artifactId>commons-logging</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

Reply:
(y) I think this dependency is more lightweight.